server/rating.js

```diff
@@ -0,0 +1,49 @@
+import express from "express";
+import fetch from "node-fetch";
+
+export const displayRating = express.Router();
+
+async function getRatingData(id) {
+  const response = await fetch(
+    "https://plankton-app-xhkom.ondigitalocean.app/api/reviews?filters%5Bmovie%5D=" +
+      id
+  );
+  const reviewData = await response.json();
+  const data = reviewData.data;
+  return data;
+}
+
+export function roundRating(passInData) {
+  //removing value of null if it exist
+  let nullValue = null;
+  let filterData = passInData.filter((item) => item !== nullValue);
+
+  let sum = filterData.reduce(function (a, b) {
+    return a + b;
+  });
+
+  let result = sum / passInData.length;
```

Comment:
This is an averaging algorithm, i.e. the formula to calculate an average value from a list of numbers. But the function is called `roundRating()`. Rounding is something else in math. The name of the function hence doesn't really communicate what the function does very clearly.